pkg/lib/aws/s3.go

```diff
@@ -291,6 +303,23 @@ func IsS3PrefixExternal(bucket string, prefix string, region string) (bool, erro
 	return hasPrefix, nil
 }
 
+func IsS3FileExternal(bucket string, key string, region string) (bool, error) {
+	sess := session.Must(session.NewSession(&aws.Config{
+		Region: aws.String(region),
+	}))
+
+	out, err := s3.New(sess).HeadObject(&s3.HeadObjectInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(key),
+	})
+
```

Comment:
Should this match more closely the implementation for `func (c *Client) IsS3File(key string)`?